extras/embed_data.bzl

```diff
@@ -34,6 +34,8 @@ go_embed_data_dependencies()
 """
 
 def _go_embed_data_impl(ctx):
+    print("Embedding is now better handled by using rules_go's built in https://github.com/bazelbuild/rules_go/blob/master/docs/go/core/embedding.md functionality. The `bindata` rule is deprecated and will be removed in rules_go version 0.35.")
```

Comment:
This link doesn't actually say anything about how to embed data. Is there any documentation for the official way to do this?